aiocache/backends/valkey.py

```diff
@@ -194,10 +201,12 @@ class ValkeyCache(ValkeyBackend):
 
     def __init__(
         self,
-        client: GlideClient,
+        client: Optional[GlideClient] = None,
         serializer: Optional["BaseSerializer"] = None,
         namespace: str = "",
         key_builder: Callable[[str, str], str] = lambda k, ns: f"{ns}:{k}" if ns else k,
+        backend: type[GlideClient] = GlideClient,
+        config: GlideClientConfiguration = None,
         **kwargs: Any,
```

Comment:
I find this a bit awkward and confusing. I think I'd rather make the config required and simplify it to:
```suggestion
        config: GlideClientConfiguration,
        serializer: Optional["BaseSerializer"] = None,
        *,
        namespace: str = "",
        key_builder: Callable[[str, str], str] = lambda k, ns: f"{ns}:{k}" if ns else k,
        **kwargs: Any,
```